website.Rmd

```diff
@@ -2,17 +2,189 @@
 
 ```{r, echo = FALSE}
 source("common.R")
-status("drafting")
+status("polishing")
 ```
 
 ## Introduction
 
+At this point, we've discussed many ways to document your package:
+
+-   Function documentation or, more generally, help topics (@sec-man).
+
+-   Documentation of datasets (@sec-documenting-data).
+
+-   Vignettes (and articles) (@sec-vignettes).
+
+-   README and NEWS (@sec-other-markdown).
+
+Wouldn't it be divine if all of that somehow got bundled up together into a beautiful website for your package?
+The [pkgdown package](https://pkgdown.r-lib.org) is meant to provide exactly this magic and that is the topic of this chapter.
+
+## Initiate a site
+
+Assuming your package has a valid structure, pkgdown should be able to make a website for it!
```

Comment:
I think somewhere we need to reinforce that the default display is pretty good, so you can be up and running with a decent website in ~5 minutes at a minimum. I think a lot of people putting off making a website because they think it's going to be a lot of work.

Reply:
I very strongly agree with this sentiment! I kept putting off "learning pkgdown" and then it turned out it was absolutely painless.

Reply:
ok added a tip about this